python/__init__.py

```diff
@@ -0,0 +1,405 @@
+import functools
+import os
+import sysconfig
+import subprocess
+import tempfile
+from pathlib import Path
+
+from triton.common.backend import BaseBackend, register_backend
+from triton.compiler.make_launcher import make_so_cache_key
+from triton.runtime.cache import get_cache_manager
+from triton.runtime.jit import version_key
+
+
+def _get_triton_shared_opt_path() -> str:
+    path = os.getenv("TRITON_SHARED_OPT_PATH", "")
+    if path == "":
+        assert Exception("TRITON_SHARED_OPT_PATH is not set.")
```

Comment:
this should be `raise`

Reply:
Thanks! Yes it should be `raise`. Updated in the latest commit.